src/Examples/Advanced/CustomProperties/custom_physical_property_packages.ipynb

```diff
@@ -0,0 +1,1329 @@
+      "enth_mol : Size=1\n",
+      "    Key  : Value\n",
+      "    None : -22169.95123146947\n"
+     ]
+    }
+   ],
+   "source": [
+    "m.fs.state[0].enth_mol.display()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We should hopefully see a value of -22170 $J/mol$, meaning our property package appears to be working.\n",
+    "\n",
+    "However, whilst the code ran smoothly and we managed to solve the model, the is still the possibility that an error slipped in somewhere. There are many things we could do to verify and validate our model, but a simple place to start is to check that all the units of measurement in the model make sense. When we declared all the variables in our model, we assigned them units of measurement so we would expect that the units should be consistent in all our model equations.\n",
+    "\n",
+    "Pyomo contains a useful tool for automatically verifying that the units of measurement are consistent, so lets import it here and run it over our model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyomo.util.check_units import assert_units_consistent\n",
+    "\n",
+    "assert_units_consistent(m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If all went according to plan, the `assert_units_consistent` method will return nothing, indicating that our units are consistent, and thus that the first step in verifying our model is complete. If  our units were not consistent, we would have seen an error telling us that the units of our model were not consistent, and a pointer to the offending constraint and the inconsistent units (Pyomo only return the first error encountered, so fixing the first error may reveal another)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Concluding Remarks\n",
+    "\n",
+    "The above example has hopefully introduced you to the basic requirements for creating your own custom physical property packages. However, it is probably clear that it requires a significant amount of effort to write your own property packages, thus users are encouraged to look into the IDAES Generic Properties Framework if they are not already familiar with this.\n",
+    "\n",
+    "The  IDAES Generic Properties Framework is designed to automatically generate user-defined property packages for common applications based on a single configuration file. User provide a list of phase and components of interest, and select fro ma library of common forms for properties of interest, and the Generic Properties Framework then does the hard work of assembling the necessary code to construct the desired model.\n",
+    "\n",
+    "To demonstrate the application of the Generic Properties Framework, an equivalent property package to the one we wrote above can be generated using the following configuration dict."
```

Comment:
Is the application of the Generic Properties Framework fully presented in the final code cell? I see that a config dictionary is setup. Is there anything else that was meant to be shown after that in the code?

Reply:
I was not intending to show anything else - just how to create the config dict. Docs for the generic properties will be else where.